src/styles/blocks/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  display: flex;
+  object-fit: contain;
+  width: 100%;
+  height: 174px;
+
+  padding: 0 3px 40px;
+  margin: 0;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+
+  padding-bottom: 4px;
```

Comment:
As for me it's more logic to set margins to set distance between elements, I mean paddings - increases element size, so if there no need to increase it (for example clickable zone), more logic to use margins. But it's not critical, just a suggestion